set.mm

```diff
@@ -202875,51 +202911,66 @@ concepts associated with those structures (product, substructure...)
     catcocl.x $e |- ( ph -> X e. B ) $.
     catcocl.y $e |- ( ph -> Y e. B ) $.
     catcocl.z $e |- ( ph -> Z e. B ) $.
-    catcocl.f $e |- ( ph -> F e. ( X H Y ) ) $.
-    catcocl.g $e |- ( ph -> G e. ( Y H Z ) ) $.
-    $( Closure of a composition arrow.  (Contributed by Mario Carneiro,
-       2-Jan-2017.) $)
-    catcocl $p |- ( ph -> ( G ( <. X , Y >. .x. Z ) F ) e. ( X H Z ) ) $=
-      ( co vg vf vx vy vz vv vw cv cop wcel wral ccat wceq wrex iscat ibi simpl
-      wa 2ralimi adantl ralimi adantr ad2antrr ad3antrrr simpllr simplr oveq12d
-      3syl eleqtrrd simpr simp-5r simp-4r opeq12d oveq123d eleq12d rspcimdv mpd
-      rspcdv ) AUAUHZUBUHZUCUHZUDUHZUIZUEUHZDTZTZWAWDGTZUJZUAWBWDGTZUKZUBWAWBGT
-      ZUKZUEBUKZUDBUKZUCBUKZFEHIUIZJDTZTZHJGTZUJZACULUJZVSVTWBWAUIWADTTVTUMUBWB
-      WAGTUKVTVSWAWAUIWBDTTVTUMUBWKUKURUDBUKUAWAWAGTUNZWHUFUHZVSWBWDUIUGUHZDTTV
-      TWCXDDTTXCWFWAWDUIXDDTTUMUFWDXDGTUKUGBUKZURZUAWIUKUBWKUKZUEBUKUDBUKZURZUC
-      BUKZWONXAXJUCUDUEUGBCDUBUAUFGULKLMUOUPXIWNUCBXHWNXBXGWLUDUEBBXFWHUBUAWKWI
-      WHXEUQUSUSUTVAVHAWNWTUCHBOAWAHUMZURZWMWTUDIBAIBUJXKPVBXLWBIUMZURZWLWTUEJB
-      AJBUJXKXMQVCXNWDJUMZURZWJWTUBEWKXPEHIGTZWKAEXQUJXKXMXORVDXPWAHWBIGAXKXMXO
-      VEXLXMXOVFZVGVIXPVTEUMZURZWHWTUAFWIXPFWIUJXSXPFIJGTZWIAFYAUJXKXMXOSVDXPWB
-      IWDJGXRXNXOVJVGVIVBXTVSFUMZURZWFWRWGWSYCVSFVTEWEWQYCWCWPWDJDYCWAHWBIAXKXM
-      XOXSYBVKZXLXMXOXSYBVLVMXNXOXSYBVEZVGXTYBVJXPXSYBVFVNYCWAHWDJGYDYEVGVOVRVP
-      VPVPVPVQ $.
-
-    catass.w $e |- ( ph -> W e. B ) $.
-    catass.g $e |- ( ph -> K e. ( Z H W ) ) $.
-    $( Associativity of composition in a category.  (Contributed by Mario
-       Carneiro, 2-Jan-2017.) $)
-    catass $p |- ( ph ->
+    ${
+      catcocl.f $e |- ( ph -> F e. ( X H Y ) ) $.
+      catcocl.g $e |- ( ph -> G e. ( Y H Z ) ) $.
+      $( Closure of a composition arrow.  (Contributed by Mario Carneiro,
+         2-Jan-2017.) $)
+      catcocl $p |- ( ph -> ( G ( <. X , Y >. .x. Z ) F ) e. ( X H Z ) ) $=
```

Comment:
What has changed in the proofs of ~catcocl and ~catass (apart from indentation) ? It looks like the compressed proofs are slightly different (e.g. `...GVOVPVQVQVQVQVR $.` versus `...TWCVTVTVTVTWCVTWD $.`).